libs/ngx-charts-on-fhir/src/lib/fhir-mappers/observation/blood-pressure-mapper.service.spec.ts

```diff
@@ -68,8 +68,8 @@ describe('BloodPressureMapper', () => {
         ],
       };
       expect(mapper.map(observation).annotations).toEqual([
-        jasmine.objectContaining({ label: { content: 'Systolic Blood Pressure Reference Range' } }),
-        jasmine.objectContaining({ label: { content: 'Diastolic Blood Pressure Reference Range' } }),
+        jasmine.objectContaining({ label: { content: 'Systolic Blood Pressure (Clinic) Reference Range' } }),
+        jasmine.objectContaining({ label: { content: 'Diastolic Blood Pressure (Clinic) Reference Range' } }),
```

Comment:
why is this expecting (Clinic) to show up in the reference range label? We don't want separate ref ranges for Home/Clinic. There should only be one reference range for Systolic and one for Diastolic.